docs/training/apis-howto/overview.rst

```diff
@@ -1,32 +1,33 @@
-###############
- Training APIs
-###############
+#####################
+ Training API Guides
+#####################
 
-This section describes how to use the training APIs and contains the API reference information.
+The Training API Guides describe how to use the Determined training APIs. 
+Each API guide contains a link to its corresponding API reference.
 
-Determined leverages specific APIs for each Deep Learning framework. In general, users convert their
+Determined leverages specific APIs for each Deep Learning framework. In general, you convert your
```

Comment:
This is out-of-date:

> Determined leverages specific APIs for each Deep Learning framework. In general, you convert your existing training code by subclassing a Trial class

Reason: it's describing the Trial APIs specifically, but this statement ignores the existence of the Core API.  Also, the usefulness of presenting the information this way may be limited to the near future.

Reply:
### Current state
- Core API
- High-level APIs
    - Trial APIs
        - PyTorchTrial
        - DeepSpeedTrial
        - *TFKerasTrial
        - *EstimatorTrial

(* = "destined for destruction")

### Future state:
- Core API
- High-level APIs
    - **Keras integration
    - **PytorchLightning integration
    - **Mosaic integration
    - Trial APIs
        - PyTorchTrial
        - DeepSpeedTrial (maybe?  not sure if we'll keep this one)

(** = "things Ryan expects we have in the future)

Reply:
created [Docs Jira issue](https://determinedai.atlassian.net/browse/DOCS-1?atlOrigin=eyJpIjoiZWQ4Y2NlNjk5MWY1NDJlY2EyNWIwMTVhYzIwYTc5MDAiLCJwIjoiaiJ9) to fix this